### DISCUSSION TITLE
Signing out Logged In Users

### DISCUSSION THREAD

**Discussion by sharktankful** [AsifArmanRahman/firebase-rest-api#24] [Q&A]
I'm having an issue trying to figure out how to sign out users that have logged in. Do you need their idToken? If so, how then would I remove the token?

**Comment by pclass-isolated** [comment]
@sharktankful  - I need to sign out a logged in user as well. I don't see a method in the docs to call. Did you ever figure this out? Thanks.

**Comment by AsifArmanRahman** [comment]
The simplest way would be to delete the token from the session, or whatever you're using to hold token. Firebase by default does not provide any sign-out API endpoints.
